src/settings/project/mod.rs

```diff
@@ -355,5 +358,40 @@ id = "0f2ac74b498b48028cb68387c421e279"
     Ok(())
 }
 
+fn check_for_duplicate_names(manifest: &Manifest) -> Result<(), failure::Error> {
+    let mut names: Vec<String> = Vec::new();
+    let mut duplicate_names: Vec<String> = Vec::new();
+    names.push(manifest.name.to_string());
+    match &manifest.env {
+        Some(environments) => {
+            for (_, environment) in environments.iter() {
+                match &environment.name {
+                    Some(name) => {
+                        if names.contains(name) && !duplicate_names.contains(name) {
+                            duplicate_names.push(name.to_string());
+                        }
+                        names.push(name.to_string());
```

Comment:
should this be in an else? it likely won't matter in the long run, but this vec will effectively end up including the duplicates.

Reply:
yes